align/pnr/toplevel.py

```diff
@@ -497,6 +498,11 @@ def gen_leaf_map(*, DB, gui):
                                 ctn = instance['concrete_template_name']
                                 if ctn in leaves:
                                     atns[atn].add((ctn, r2wh(leaves[ctn]['bbox'])))
+=======
+def process_placements(*, DB, verilog_d, gui, lambda_coeff, scale_factor, reference_placement_verilog_json, opath):
+    leaf_map = gen_leaf_map(DB=DB, gui=gui)
+    tagged_bboxes = defaultdict(dict)
+>>>>>>> master
```

Comment:
@stevenmburns There is a merge conflict to the master branch. Do you know how to solve it?

Reply:
@854768750 resolved the merge conflict but still, there are two failures.
1. tests/pdk/finfet/test_circuits.py::test_cmp
2. tests/pdk/finfet/test_circuits.py::test_ro_1

Reply:
@soneryaldiz 
I used `pytest -v test_circuits.py -k test_ro --runnightly` but got `TypeError`.
I also tried `schematic2layout.py` but there was no error.
How can I know where is wrong?